src/utils/symlink.ts

```diff
@@ -42,24 +42,25 @@ export function createSymlinks(
   const baseVersionName = path.basename(versionFilePath);
   const packageDir = path.dirname(versionFilePath);
 
-  // link latest, but only if the new version is "newer"
+  // link latest.json and {major}.json, but only if the new version is "newer"
   if (
     parsedOldVersion &&
     !versionGreaterOrEqualThan(parsedNewVersion, parsedOldVersion)
   ) {
     logger.warn(
-      `Not updating the latest version file: current version is "${oldVersion}", new version is "${newVersion}"`
+      `Not updating the latest version file: current version is "${oldVersion}", new version is "${newVersion}"\n`,
+      `Not updating the major version file: current version is "${oldVersion}", new version is "${newVersion}"`
     );
   } else {
     logger.debug(
-      `Changing symlink for "latest.json" from version "${oldVersion}" to "${newVersion}"`
+      `Changing symlink for "latest.json" from version "${oldVersion}" to "${newVersion}"\n`,
+      `Changing symlink for "{major}.json" from version "${oldVersion}" to "${newVersion}"`
```

Comment:
Same as above. Also please move this new log line below, right before the actual command call.